helm/lint.go

```diff
@@ -91,11 +140,21 @@ func Lint(ctx *ankh.ExecutionContext, helmOutput string, ankhFile ankh.AnkhFile)
 			continue
 		}
 
+		objects = append(objects, obj)
+
 		errors := LintObject(ctx, ankhFile, obj)
 		if len(errors) > 0 {
 			allErrors = append(allErrors, errors...)
 		}
 	}
+
+	if ctx.Pedantic {
+		errors := PedanticLint(ctx, ankhFile, objects)
```

Comment:
can we push PedanticLint into `LintObject`?

Reply:
Some of our lint conditions care about context above the object level (for example, verifying that a service exists in the chart)

Reply:
let's omit chart-level linting in ankh